c_emulator/riscv_callbacks.h

```diff
@@ -18,7 +18,7 @@ unit csr_full_write_callback(const_sail_string csr_name, unsigned reg,
                              uint64_t value);
 unit csr_full_read_callback(const_sail_string csr_name, unsigned reg,
                             uint64_t value);
-unit vreg_write_callback(unsigned reg, lbits value);
+unit vreg_write_callback(unsigned reg, lbits value, uint64_t vlen);
```

Comment:
```suggestion
unit vreg_write_callback(unsigned reg, lbits value, uint64_t vlenb);
```
To make it clear that this is vlen in bytes for any other callback implementations.

Reply:
Looks like it was changed in `riscv_callbacks.cpp` but not `riscv_callbacks.h`.

Reply:
Ups, my bad, thanks!